### PR TITLE
feat(client): show OS-specific modifier key instead of Ctrl/⌘

### PR DESCRIPTION
**Pull request by maraf**
The keyboard shortcut hint next to the Send button showed "Ctrl/⌘+Enter" regardless of the user's platform, which felt generic and slightly cluttered.

This adds a small `platform.ts` utility that detects macOS via `navigator.platform` and exports a `modifierKey` constant. The hint now renders the actual modifier for the current OS -- `⌘+Enter` on Mac, `Ctrl+Enter` everywhere else.

### CODE REVIEW

File: src/SharedSpaces.Client/src/lib/platform.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const isMac =
+  typeof navigator !== 'undefined' &&
+  /Mac|iPhone|iPad|iPod/.test(navigator.platform);
```
[the remark this answers is not in the document]
Comment (maraf) [reply src/SharedSpaces.Client/src/lib/platform.ts:3]: Good call. Added `platform.test.ts` covering Mac/iOS platforms (returns `⌘`), Windows/Linux (returns `Ctrl`), and the edge case where `navigator` is undefined. All 9 tests pass.
